src/components/tile/tile.component.tsx

```diff
@@ -35,15 +35,21 @@ export interface TileProps extends SpaceProps, WidthProps {
     | "info";
   /** Sets the level of roundness of the corners, "default" is 8px and "large" is 16px */
   roundness?: "default" | "large";
+  /**
+   * Set a percentage-based height for the whole Tile component, relative to its parent.
+   * If unset or zero, this will default to 100%.
```

Comment:
**question**: It seems we don't have a default value for `height`, which I assume is because we don't want the `Tile` completely filling its parent container by default?

Reply:
Yes that was my thinking behind it 👍